packages/react-core/src/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -48,52 +48,24 @@ export const BackgroundImage: React.FunctionComponent<BackgroundImageProps> = ({
   className = '',
   src,
   ...props
-}: BackgroundImageProps) => {
-  let srcMap = src;
-  // Default string value to handle all sizes
-  if (typeof src === 'string') {
-    srcMap = {
-      [BackgroundImageSrc.xs]: src,
-      [BackgroundImageSrc.xs2x]: src,
-      [BackgroundImageSrc.sm]: src,
-      [BackgroundImageSrc.sm2x]: src,
-      [BackgroundImageSrc.lg]: src,
-      [BackgroundImageSrc.filter]: '' // unused
-    };
-  }
-
-  // Build stylesheet string based on cssVariables
-  let cssSheet = '';
-  (Object.keys(cssVariables) as [keyof typeof srcMap]).forEach(size => {
-    cssSheet += `${cssVariables[size as keyof typeof cssVariables]}: url('${srcMap[size]}');`;
-  });
-
-  // Create emotion stylesheet to inject new css
-  const bgStyles = StyleSheet.create({
-    bgOverrides: `&.pf-c-background-image {
-      ${cssSheet}
-    }`
-  });
-
-  return (
-    <div className={css(styles.backgroundImage, bgStyles.bgOverrides, className)} {...props}>
-      <svg xmlns="http://www.w3.org/2000/svg" className="pf-c-background-image__filter" width="0" height="0">
-        <filter id="image_overlay">
-          <feColorMatrix
-            type="matrix"
-            values="1 0 0 0 0
-            1 0 0 0 0
-            1 0 0 0 0
-            0 0 0 1 0"
-          />
-          <feComponentTransfer colorInterpolationFilters="sRGB" result="duotone">
-            <feFuncR type="table" tableValues="0.086274509803922 0.43921568627451" />
-            <feFuncG type="table" tableValues="0.086274509803922 0.43921568627451" />
-            <feFuncB type="table" tableValues="0.086274509803922 0.43921568627451" />
-            <feFuncA type="table" tableValues="0 1" />
-          </feComponentTransfer>
-        </filter>
-      </svg>
-    </div>
-  );
-};
+}: BackgroundImageProps) => (
+  <div className={css(styles.backgroundImage, className)} {...props}>
```

Comment:
Did we introduce a breaking change here? 

The `cssVariables` const and exported `BackgroundImageSrc` enum appear to be unused.

This was in place was so devs didn't need to provide a src path. I recall this was difficult to nail down, but maybe that's available via core CSS now?

Reply:
Yep, this is a problem that needs addressing.

This whole PR is a breaking change, though, so I'll find a solution and push it once the rest looks good.

Reply:
I found inline styles works great: no need for emotion.